backend/src/modules/transfer/utils/createBagitFiles.ts

```diff
@@ -0,0 +1,23 @@
+import type { TransferZod } from "@/modules/transfer/entities";
+import type { FileMetadataZodType } from "@/modules/filelist/schemas";
+
+export const createBagitFiles = (
+	files: TransferZod["metadata"]["files"],
+): { bagit: Buffer; manifest: Buffer } => {
+	// Create the content for bagit.txt
+	const bagitContent = "BagIt-Version: 1.0\nTag-File-Character-Encoding: UTF-8\n";
+
+	// Create the content for the manifest file
+	let manifestContent = "";
+	Object.entries(files).forEach(([_, fileArray]) => {
+		fileArray.forEach((file: FileMetadataZodType) => {
```

Comment:
Should we include a verification on the contents of `file`? I was able to remove all required fields and the process ran with no issues.

Reply:
I think its fine to leave. It is type safe so it should complain if you're not passing the right type and validation of files is a separate function that would come before this is used.